Dockerfile

```
@@ -1,15 +1,39 @@
-COPY requirements.txt /app/
-COPY .env /app/
+# Install system dependencies 
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    curl \
+    ffmpeg \
+    && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /app
+# Install Poetry
+RUN curl -sSL https://install.python-poetry.org | python3 -
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)